samples/drivers/mbox/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -6,13 +6,17 @@
 
 / {
 	soc {
-		mbox: mbox@2a000 {
-			compatible = "nordic,mbox-nrf-ipc";
-			reg = <0x2a000 0x1000>;
-			tx-mask = <0x0000ffff>;
-			rx-mask = <0x0000ffff>;
-			interrupts = <42 NRF_DEFAULT_IRQ_PRIORITY>;
-			status = "okay";
+		peripheral@50000000 {
+			/delete-node/ ipc@2a000;
```

Comment:
Is it necessary to `delete-node`? Wouldn't it be enough to set its `status = "disabled";`?

Reply:
it's not necessary but the generated DT will still have the IPC node and that is confusing if someone looks at that. Better deleting it.

Reply:
If we move mbox description from sample to soc's DT, would it be alongside ipc? Just one of them `"okay"` at a time?

Reply:
>  would it be alongside ipc?

Correct.

> Just one of them "okay" at a time?

Yes, the user should "okay" only one of the two.